### PR TITLE
Set publisher expectations to 1.50 - 2

### PR DESCRIPTION
**Pull request by davidfischer**

### CODE REVIEW

File: ethicalads-theme/templates/ea/publishers.html [hunk 1]
```diff
@@ -151,7 +151,7 @@ <h4 class="font-weight-bold" id="faq-get-started">How do I get started?</h4>
       <h4 class="font-weight-bold" id="faq-how-much">How much will I make?</h4>
       <p>
         These numbers will vary based on your audience and topics that you write about.
-        Generally we are able to offer around $2-2.50 per 1,000 pageviews (CPM),
+        Generally publishers see around $1.50-2.00 per 1,000 pageviews (CPM),
```
Comment (ericholscher) [inline ethicalads-theme/templates/ea/publishers.html:154]: I think let's just say "around $2" here, and expand a bit more in the FAQ?

File: content/pages/publisher-faq.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Yes, please read our full [Publisher Policy](/publisher-policy/#ad-display) on a
 ## How much will I make?
 
 These numbers will vary based on your audience and topics that you write about.
-Generally we are able to offer around $2-2.50 per 1,000 pageviews (CPM),
+Generally publishers see around $1.50-2.00 per 1,000 pageviews (CPM),
 assuming a majority of US and European traffic.
```
Comment (ericholscher) [inline content/pages/publisher-faq.md:39]: I think we could cover more about how we price differently for geos & topics here, with a bit more detail?

**Review by ericholscher — APPROVED** [review]

File: content/pages/publisher-faq.md (hunk 2 above)
Comment (davidfischer) [reply content/pages/publisher-faq.md:39]: Sounds good. The FAQ is a place where we can have more details.

**Comment by davidfischer** [comment]
I added a few more details around publisher expectations specifically around geo, topic, and CPC.

**Review by ericholscher — APPROVED** [review]
Changes look good 👍

**Pull request merged**